src/palace/manager/api/circulation/dispatcher.py

```diff
@@ -194,6 +194,13 @@ def borrow(
             # with a collection that this library doesn't have access to.
             raise NoLicenses()
 
+        if licensepool.unlimited_type and not licensepool.active_status:
+            # The license pool is of unlimited type but is not currently active.
+            # This means that although there are no limits on the number of
+            # simultaneous users, the licensepool is not currently available
+            # for borrowing for some reason.
+            raise NoLicenses()
```

Comment:
We enforce that unlimited licenses need to be active here, otherwise they would be filtered out of search, but still able to be checked out by identifier.